scss/components/_welcome.scss

```diff
@@ -0,0 +1,225 @@
+@use '../abstracts' as *;
+
+.welcome {
+    position: relative;
+    @include flexbox(column, center, center);
+    margin: 0 0 2rem 0;
+    text-align: center;
+    width: 100%;
+
+    &__box, &__description {
+        padding: 1rem;
+    }
+
+    &__box {
+        @include flexbox(column, center, center);
+        margin-top: 5rem;
+    }
+
+    &__title {
+        font-size: 3rem;
+        width: 75%;
+    }
+
+    &__geeks-span {
+        color: $primary-color;
+    }
+
+    &__description {
+        margin-top: 1rem;
+        font-size: 1.5rem;
+    }
+
+
+    &__btn-box {
+        margin: 1.5rem;
+    }
+
+    &__button {
+        margin: 0 0.5rem;
+        padding: 1.5rem 2.5rem;
+        cursor: pointer;
+
+        &--primary {
+            @include btn-primary();
+        }
+
+        &--secondary {
+            @include btn-secondary();
+        } 
+    }
+
+    &__cart-icon {
+        padding-right: 0.5rem;
+    }
+
+    &__preview-icon {
+        margin-left: 0.5rem;
+    }
+
+    &__rating {
+        @include flexbox();
+        margin: 1.5rem auto;
+    
+        &-score, &-description {
```

Comment:
is it modifier or just a element name? :thinking:

Reply:
They are both elements inside welcome__rating element